### PR TITLE
Commercial: improvements to Fabric Video

### PR DESCRIPTION
**Pull request by regiskuckaertz**
As documented in [the Trello card](https://trello.com/c/6rsekQXD/61-fabric-template-issues)

_Do not merge until I have made sure every use case is covered with Tijana_

cc @guardian/commercial-dev

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
Now approved by the rich media team 😎

File: static/src/javascripts/projects/common/modules/commercial/creatives/fabric-video.js [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
     'text!common/views/commercial/creatives/fabric-video.html'
-], function (qwery, fastdom, detect, template, fabricVideoStr) {
+], function (qwery, bonzo, fastdom, detect, template, addTrackingPixel, fabricVideoStr) {
     var fabricVideoTpl;
 
     return FabricVideo;
 
     function FabricVideo(adSlot, params) {
         var isUpdating = false;
+        var isMobile = detect.isBreakpoint({ max: 'phablet' });
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/commercial/creatives/fabric-video.js]: `var isMobileOrTablet` ?

File: static/src/stylesheets/module/commercial/creatives/_fabric-video.scss [hunk 2]
```diff
@@ -2,27 +2,34 @@
...
-    .creative__alt,
-    .creative__layer,
-    .creative__slide {
-        position: absolute;
-        top: 0;
-        left: 0;
-        width: 100%;
-        height: 100%;
+    .creative__video--left {
+        object-position: left;
```
Comment (rich-nguyen) [inline static/src/stylesheets/module/commercial/creatives/_fabric-video.scss]: no ie support?

**Comment by rich-nguyen** [comment]
👍

File: static/src/javascripts/projects/common/modules/commercial/creatives/fabric-video.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/creatives/fabric-video.js]: Yes that is counter-intuitive. It really is to detect the size of the viewport here. Thanks for spotting it!

File: static/src/stylesheets/module/commercial/creatives/_fabric-video.scss (hunk 2 above)
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/creatives/_fabric-video.scss]: Alas, nope, not even `object-fit`, which means the embedded element will probably get resized on IE.

**Pull request merged**